### DISCUSSION TITLE
How not to convert time when using open_mfdataset?

### DISCUSSION THREAD

**Discussion by loganchen39** [pydata/xarray#7535] [Q&A] [answered]
I'm using open_mfdataset to read multiple daily sea-surface-height netCDF files, it converts the variable "double time_mjd(time)", which has an attribute "time_mjd:units = "days since 1858-11-17 00:00:00 UTC" ;" to numpy datetime64[ns]. How to avoid the conversion? I only need the original "double" values. 
Meanwhile there's another variable "short sla(time)" which has attributes of scale_factor, for this one I'd like to convert it to float type which open_mfdataset does, so I wanna keep it this way. Thanks,

**Comment by spencerkclark** [comment, marked as answer]
I think passing `decode_times=False` to `open_mfdataset` should accomplish what you are looking for.

**Reply by loganchen39** [reply]
Yes, I just found the parameter from open_dataset, it worked, thanks,
